### PR TITLE
ROT-348: Add links to footer images

### PR DESCRIPTION
**Pull request by ccarney96**

### CODE REVIEW

**Review by hannah-whelan — APPROVED** [review]

File: src/components/Footer/Footer.tsx [hunk 1]
```diff
@@ -26,18 +26,22 @@ const Footer = () => (
             <p
               className="
                 font-bold mb-4 flex text-center justify-center md:justify-start
-            " 
+            "
             >
               Powered by
             </p>
-            <img src={RTLogo} alt="Rotherham Titans Rugby" className="w-16 mx-auto md:mx-0" />
+            <a href="https://www.titans-rugby.com/">
```
Comment (hannah-whelan) [inline src/components/Footer/Footer.tsx:33]: Would it be worth using target="_blank" here so that clicking on the links opens into a new tab?

Comment (ccarney96) [reply src/components/Footer/Footer.tsx:33]: Yes, good shout, will change it :)

File: src/components/Footer/Footer.tsx [hunk 2]
```diff
@@ -30,7 +30,7 @@ const Footer = () => (
             >
               Powered by
             </p>
-            <a href="https://www.titans-rugby.com/">
+            <a href="https://www.titans-rugby.com/" target="_blank" rel="noreferrer">
```
Comment (hannah-whelan) [inline src/components/Footer/Footer.tsx:33]: Do we need the "noreferrer"? I'd have thought there's no issue with allowing analytics to see if traffic coming to RT or AND websites are from Cookie's Khaw?

Comment (ccarney96) [reply src/components/Footer/Footer.tsx:33]: ESLint didn't like it without a noreferrer as it's vulnerable on older browsers.

**Review by tom-downes-and — APPROVED** [review]

**Pull request merged**